docs/_frontend/src/models/Sidebar/Sidebar.js

```diff
@@ -58,8 +58,11 @@ const Sidebar = ({ meta, params, onSubmit }) => {
           }}
         >
           <label>Sort By:</label>
-          <Radio value="date">Date</Radio>
-          <Radio value="views">Views</Radio>
+          <div className={styles.radioContainers}>
```

Comment:
Let's apply prettier to js and css files.

Reply:
should I run prettier with single quote parameter?